test/functional/apps/management/_scripted_fields.js

```diff
@@ -56,13 +56,13 @@ bdd.describe('creating and using Lucence expression scripted fields', function d
     await PageObjects.common.navigateToApp('discover');
     await PageObjects.common.debug('setAbsoluteRange (' + fromTime + ') to (' + toTime + ')');
     await PageObjects.header.setAbsoluteRange(fromTime, toTime);
-    await PageObjects.header.isGlobalLoadingIndicatorHidden();
+    await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
I really think these waitUntilLoadingHasFinished calls should be in the page object methods instead of the tests.  For example, at the end of setAbsoluteRange it should wait.  That would focus the waiting call right after the clicks that cause the loading indicator to appear.